README.md

```diff
@@ -85,6 +85,35 @@ npm start
 gulp watch
 ```
 
+### Working with npm link
+
+```bash
+# in ng2-json-editor
+
+# do initial build into dist
+npm run build
+
+# copy package.json to dist
+npm run copy
```

Comment:
`package.json` is needed for linking inside dist folder. Otherwise, it will link from the nearest folder that includes one.